docs/content/tutorials/kic/deploy-with-kic.md

```diff
@@ -214,13 +218,18 @@ deployments are not supported. {{</note>}}
 ### Enable with Manifests
 If you are installing NGINX Plus Ingress Controller with manifests follow the [Install with Manifests](#install-with-manifests) instructions and make the following changes to the NGINX Plus Ingress Controller Pod spec:
 
-- Add the following annotation to the NGINX Plus Ingress Controller Pod spec:
+- Add the following label to the NGINX Plus Ingress Controller Pod spec:
 
     ```bash
     nsm.nginx.com/enable-egress: "true"
     ```
 
-  This annotation prevents automatic injection of the sidecar proxy and configures the NGINX Plus Ingress Controller as the egress endpoint of the mesh.
+  This label prevents automatic injection of the sidecar proxy and configures the NGINX Plus Ingress Controller as the egress endpoint of the mesh.
+
+  {{<note>}}
+  In NGINX Service Mesh versions prior to v1.7.0, the `nsm.nginx.com/enable-egress: "true"` annotation was used instead of a label.
+  Support for this annotation will be removed in a future release.
+  {{</note>}}
```

Comment:
Screenshot of changes from local testing.
![Screen Shot 2022-12-30 at 11 50 12 AM](https://user-images.githubusercontent.com/50845045/210107594-2bfd5675-ff26-4219-b340-7c1fc57c4cb0.png)